<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@400;700&display=swap">
    <title>Nica Gutu - Portfolio</title>
    <style>
        body {
            font-family: 'Raleway', serif;
            margin: 0;
            padding: 0;
            background-color: #EBF2F8; 
            color: #606060; 
            transition: background-color 0.3s ease;
        }

        header {
            background-color: #fff; 
            color: #36486b; 
            padding: 10px 10px;
            text-align: center;
            position: relative;
            transition: background-color 0.3s ease;
        }

        header img {
            width: 160px;
            height: 160px;
            border: 5px solid #becbd6; 
            border-radius: 50%;
            position: absolute;
            top: 20px;
            left: 50%;
            transform: translateX(-50%);
            transition: transform 0.3s ease;
        }

        header img:hover {
            transform: translateX(-50%) scale(1.1);
        }

        header h1 {
            margin: 180px 0 10px;
            font-size: 3em;
            color: #36486b; 
            transition: color 0.3s ease;
        }

        header p {
            font-size: 1.1em;
            margin: 10px 0;
            text-align: center;
        }

        nav {
            display: flex;
            justify-content: center;
            background-color: #fff; 
            padding: 10px;
        }

        nav a {
            color: #606060;
            padding: 10px 20px;
            text-decoration: none;
            text-transform: uppercase;
            font-weight: bold;
            transition: background-color 0.3s ease, color 0.3s ease;
        }

        nav a:hover {
            background-color: #8b94a5;
            color: #fff;
        }
        
        nav a.active {
            background-color: #8b94a5; 
            color: #fff;
        }       

        section {
            display: none;
            padding: 20px;
            margin: 20px;
            background-color: #fff;
            border-radius: 8px;
            box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
            transition: opacity 0.5s ease, transform 0.5s ease;
            transform: translateY(20px);
            opacity: 0;
        }

        section.active {
            display: block;
            opacity: 1;
            transform: translateY(0);
        }

        footer {
            background-color: #fff;
            color: #606060; 
            text-align: center;
            padding: 1px;
            position: fixed;
            bottom: 0;
            width: 100%;
            font-size: 12px;
            display: flex;
        justify-content: center; 
        align-items: center;
        flex-direction: column; 
        }

        footer a {
            color: #687387; 
            text-decoration: none;
            font-weight: bold;
        }

        footer a:hover {
            text-decoration: underline;
        }


        .timeline-item, .project-item, .publications-item, .skills-item, .conferences-item {
            margin-bottom: 20px;
            padding: 15px;
            border-left: 4px solid #8b94a5; 
            background-color: #fff;
            border-radius: 8px;
            transition: border-color 0.3s ease, box-shadow 0.3s ease;
        }

        .timeline-item:hover, .project-item:hover, .publications-item:hover, .skills-item:hover, .conferences-item:hover {
            border-color: #687387; /* Dusty Blue */
            box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
        }

        .timeline-item h3, .project-item h3, .publications-item h3, .skills-item h3, .conferences-item h3 {
            margin-top: 0;
            color: #2D3A44; 
        }

        .timeline-item a, .project-item a, .publications-item a, .skills-item a, .conferences-item a {
            color: #687387; 
            text-decoration: none;
            font-weight: bold;
        }

        .timeline-item a:hover, .project-item a:hover, .publications-item a:hover, .skills-item a:hover, .conferences-item a:hover {
            text-decoration: underline;
        }

        p {
            text-align: justify;
        }

        #projects {
            padding: 20px;
        }

        .project-item {
            margin-bottom: 30px;
        }


        .project-item h3 {
            margin-top: 0;
        }

        .small-img {
            width: 40%;
            height: auto;
        }

        .large-img {
            width: 65%;
            height: auto;
        }

        .medium-img {
            width: 55%;
            height: auto;
        }

        @media (max-width: 768px) {
            nav {
                flex-direction: column;
                align-items: center;
            }

            nav a {
                padding: 10px;
            }
        }

        .social-icons a {
            color: #606060; 
            text-decoration: none;
            margin: 0 10px;
            font-size: 2em;
            transition: color 0.3s;
        }

        .social-icons a:hover {
            color: #0077b5;
        }
    </style>
</head>

<body>

    <header>
        <img src="NG_foto.jpeg" alt="Nica Gutu">
        <h1>Nica Gutu</h1>
        <p>Biophysicist & Computational Biologist | Breast and Bone Cancer Research | Biological rhythms | Celullar proliferation | DNA Damage Response | Pharmacodynamics</p>
    </header>

    <nav>
        <a id="nav-about" onclick="showSection('about')">About me</a>
        <a id="nav-education" onclick="showSection('education')">Education</a>
        <a id="nav-experience" onclick="showSection('experience')">Experience</a>
        <a id="nav-publications" onclick="showSection('publications')">Publications</a>
        <a id="nav-projects" onclick="showSection('projects')">Projects</a>
        <a id="nav-conferences" onclick="showSection('conferences')">Conferences</a>
        <a id="nav-skills" onclick="showSection('skills')">Skills</a>
        <a id="nav-languages" onclick="showSection('languages')">Languages</a>
        <a id="nav-contact" onclick="showSection('contact')">Contact</a>
    </nav>

    <section id="about" class="active">
        <h2>ABOUT ME</h2>
        <p>Welcome to my professional portfolio! I am Nica Gutu, a physicist with a PhD in Biophysics and Computational Biology, with 4 years of experience in breast and bone cancer research. My academic journey and research endeavors have honed my expertise in deciphering the complex mechanisms of cell proliferation and drug response, employing advanced data science, mathematical modeling, bioinformatics, and machine learning techniques.</p>

        <p>Over the past years, I have delved deeply into the cellular dynamics that drive tumor heterogeneity and treatment resistance, with a particular focus on the regulatory role of biological rhythms in cancer therapy. My research has uncovered the pivotal influence of the circadian clock on tissue growth dynamics, identifying optimal time windows for cell division and revealing how the timing of drug administration can profoundly affect therapeutic outcomes. Through a combination of live-cell imaging, time-series analysis, and integrative computational approaches, I have shed light on the nuanced interplay between DNA damage response signaling and cell proliferation, offering new perspectives on optimizing cancer treatments.</p>

        <p>I am now eager to take on new challenges, where I can apply my skills and experience!</p>
    </section>

    <section id="education">
        <h2>EDUCATION</h2>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>PhD in Biophysics – Computational Biology</h3>
                <p><em>Humboldt-Universität zu Berlin. Berlin, Germany | 02.2021 – 08.2024</em></p>
                <p>Thesis: "Deciphering chronobiological regulation of cell proliferation and drug responses: Insights from the circadian clock and p53-p21 dynamics"</p>
            </div>
        </div>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>MSc in Engineering Physics</h3>
                <p><em>Universitat Politècnica de Catalunya (UPC). Barcelona, Spain | 09.2019 – 09.2020</em></p>
                <p>Thesis in Computational Biology: "Biochemical model for the amoeboid motion of Dictyostelium discoideum cells" <a href="https://upcommons.upc.edu/handle/2117/354385" target="_blank">(link)</a></p>
            </div>
        </div>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>BSc in Physics</h3>
                <p><em>Universitat de Barcelona (UB). Barcelona, Spain | 09.2014 – 01.2019</em></p>
                <p>Thesis in Biophysics: "Morphogenesis: how can periodic patterns be generated from non-linear interactions" <a href="https://diposit.ub.edu/dspace/bitstream/2445/140319/1/GUTU%20Nica.pdf" target="_blank">(link)</a></p>
            </div>
        </div>
    </section>

    <section id="experience">
        <h2>EXPERIENCE</h2>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>Research Assistant</h3>
                <p><em>Systems Oncology Lab <a href="http://granadalab.org" target="_blank">(link)</a>, Charité Comprehensive Cancer Center. Berlin, Germany | 02.2021 – Present</em></p>
                <ul>
                    <li>Data scientist and computational biologist</li>
                    <li>Investigating cellular proliferation to predict drug response in bone and breast cancer and align therapeutic strategies with biological rhythms to enhance treatment efficacy, maximize health benefits, and minimize adverse effects.</li>
                    <li>Developing predictive models to understand the mechanisms underlying cellular responses in cancer treatment.</li>
                    <li>Utilizing time-series and data analysis, computational biology, biostatistics, bioinformatics, machine learning and image analysis techniques to analyze complex biological data from protein to tissue level from healthy and tumor cells.</li>
                    <li>Tools: Python, MATLAB, R, ilastik, ImageJ, CellProfiler, MetaXpress, Incucyte Software, inkscape</li>
                </ul>
            </div>
        </div>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>Research/Laboratory Technician</h3>
                <p><em>Soluciones Electrostáticas, S.L., Barcelona, Spain | 09.2020 – 02.2021</em></p>
                <ul>
                    <li>Provided laboratory services and consulting to address electrostatic issues in electronics, automotive, and chemical industries.</li>
                    <li>Calibrated, controlled, and repaired equipment to solve electrostatic-related problems.</li>
                    <li>Conducted audits to ensure compliance and safety for various companies.</li>
                </ul>
            </div>
        </div>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>Software Engineer</h3>
                <p><em>Accenture S.L., Barcelona, Spain | 12.2018 – 06.2019</em></p>
                <ul>
                    <li>Worked on a banking project focused on big data management and control of ATM log files using Elastic Search and HTML.</li>
                    <li>Developed solutions to handle and analyze large volumes of data generated by ATM transactions.</li>
                    <li>Ensured robust data integrity and security through efficient data processing algorithms.</li>
                </ul>
            </div>
        </div>
        <div class="timeline-item">
            <div class="dot"></div>
            <div class="content">
                <h3>Intern</h3>
                <p><em>Department of Data Protection, Universitat de Barcelona (UB), Barcelona, Spain | 06.2018 – 12.2018</em></p>
                <ul>
                    <li>Conducted data privacy audits and ensured compliance with GDPR regulations.</li>
                </ul>
            </div>
        </div>
    </section>

    <section id="publications">
        <h2>PUBLICATIONS</h2>
        <div class="project-item">
            <h3>“p53 and p21 dynamics encode single-cell DNA damage levels, fine-tuning proliferation and shaping population heterogeneity.”</h3>
            <p><strong>Gutu, N.</strong> et al. <em>Communications Biology 6, 1196 (2023)</em>. <a href="https://www.nature.com/articles/s42003-023-05585-5" target="_blank">Read more</a></p>
        </div>
        <div class="project-item">
            <h3>“Circadian Coupling Orchestrates Cell Growth”</h3>
            <p><strong>Gutu, N.</strong> et al.  Preprint in review at Nature Physics (2024). <a href="https://www.biorxiv.org/content/10.1101/2024.05.18.594797v1" target="_blank">Read more</a></p>
        </div>
        <div class="project-item">
            <h3>“Circadian period is compensated for repressor protein turnover rates in single cells”</h3>
            <p>Gabriel, C.H., <strong> Gutu N.</strong> et al. <em>PNAS, 121(34) (2024)</em>. <a href="https://www.pnas.org/doi/10.1073/pnas.2404738121" target="_blank">Read more</a></p>
        </div>
        <div class="project-item">
            <h3>“Time Matters: Understanding the Drivers of Time-of-Day Drug Sensitivity”</h3>
            <p><strong>Gutu, N. </strong>et al. (Recently Submitted)</p>
        </div>
    </section>

    <section id="projects">
        <h2>PROJECTS</h2>
        <div class="project-item">
            <h3>DNA Damage Response: Signaling Dynamics and Radiation Response Heterogeneity</h3>
            <img src="graphical_abstract_p53.png" alt="DDR_p53_p21" class="small-img">
            <p>
                The response to DNA damage and the regulation of proliferation are key cellular functions that maintain genome integrity and tissue homeostasis. Proliferation heterogeneity can strongly affect tissue development, give rise to tumor progression and drive treatment response variability. Understanding the mechanisms that regulate proliferation heterogeneity is essential for advancing our understanding of this fundamental process and developing new treatment strategies for a wide range of diseases.
            </p>
            <p>
                In this project, I investigated the mechanisms modulating individual proliferation patterns and their impact on collective proliferation heterogeneity. My study provides the first demonstration of how the individual signal parameters of the DNA damage responsive transcription factor p53 and its target p21 encode gradual information about stress levels, which in turn regulate proliferation patterns. While previous pioneering studies have shown how abrupt qualitative changes in signaling lead to alternative cell fates, this work addresses a major knowledge gap by elucidating how cells quantitatively encode different stress levels to balance outcomes within repair, arrest, or gradual proliferation responses. Collectively, these findings advances the methods to decode single cell signaling dynamics and provides a remarkable step forward in understanding non-genetic mechanisms of resistance. <a href="https://www.sys-med.de/fileadmin/Aktuelles/Newsletter/eMedium_7_Juli_2024_online.pdf#page=2" target="_blank">Read more e:Med Newsletter (pg. 2)</a> <a href="https://www.nature.com/articles/s42003-023-05585-5" target="_blank">& Communications Biology</a></p>
        </div>
        <div class="project-item">
            <h3>Circadian Rhythm and Cell Growth</h3>
            <img src="graphical_abstract_circ_cellcycle0.png" alt="Circ_CellCycle0" class="large-img">
            <p>
                At tissue level, circadian clocks play a crucial role in synchronizing rhythms, coordinating main cellular processes, and regulating thousands of genes for cellular balance. Recent works have unveiled how circadian clock dysfunctions can spur oncogenic processes, impacting tumor growth. However, understanding the precise mechanisms governing these interactions remains unknown. 
            </p>
            <p>
                To address these gaps, I employed a multifaceted approach, manipulating circadian coupling between cells while monitoring individual cells over an extended period. I combined coupled oscillators theory with long-term imaging at both the single-cell and population levels, incorporating genetic and non-genetic perturbations. Through this, I could modulate circadian synchronization among cells and explore its repercussions on population growth and its interplay with the cell cycle. Notably, this study is the first to demonstrate that the level of circadian coordination modulates cell growth, and how its disruption results in alterations in tissue growth patterns. Taken together, these findings unveil the circadian clock's timing mechanism as a crucial orchestrator of tissue growth dynamics, determining optimal time windows for cellular divisions. By addressing fundamental questions about the temporal organization of central cellular processes in tissue homeostasis and growth regulation, this study sheds light on the intricate interplay between circadian rhythms, cellular signaling, and tissue physiology in health and disease.<a href="https://www.biorxiv.org/content/10.1101/2024.05.18.594797v1" target="_blank"> Read more</a></p>
        </div>
        <div class="project-item">
            <h3>Drug Response Dynamics in Chronotherapy</h3>
            <img src="graphical_abstract_ToD.png" alt="ToD_factors" class="small-img">
            <p>
                The impact of the circadian clock on health and disease, particularly how treatment time affects responses, is well-documented. However, integrating time-of-day considerations into clinical drug administration is still very limited. This is primarily due to contradictory results on the role of the circadian clock and an incomplete understanding of the mechanisms driving time-dependent drug sensitivity. In laboratory settings, time-of-day drug sensitivity is studied by systematically testing the effects of treatments at different times of the day. These investigations have been highly insightful, showing that time-of-day sensitivity profiles vary significantly based on the specific tumor cell type and drug used. However, the numerous genetic and epigenetic differences among tumor models make it difficult to pinpoint the exact reasons behind the variations in time-of-day profiles, offering limited information on the underlying mechanistic factors. 
            </p>
            <p>
                This project addresses this knowledge gap by combining mathematical modeling with precise in-vitro experiments to elucidate the
                cellular factors governing drug sensitivity across different times of the day in human cells. The findings highlight the significant influence of three central properties of tumor cell models on the time-of-day efficacy profiles: the circadian clock, drug sensitivity profile, and cellular growth. I demonstrate that the interplay of these factors can strongly affect various aspects of the time-of-day response curves. Additionally, when combined with the experimental constraints of standard in-vitro assays, these factors can
                markedly alter the outcome response profiles. Together, my work provides a comprehensive mechanistic
                framework to disentangle the underlying cellular properties that shape time-of-day profiles, offering valuable
                insights for identifying mechanistic drives of time-of-day sensitivity and ultimately paving the way for
                optimizing treatment strategies.
                <a href="https://github.com/nica-gutu/determinants_ToD_drug_sensitivity" target="_blank"> Read more</a></p>
        </div>
    </section>

    <section id="conferences">
        <h2>CONFERENCES</h2>
        <div class="project-item">
            <h3>BIMSB Symposium: Single Cells in Focus</h3>
            <p><em>03.2024. Berlin, Germany. <strong>Scientific talk on gradual signaling of DNA damage response</strong></em></p>
            <p>Cells must accurately and quickly detect DNA damage through a set of checkpoint mechanisms that enable repair and control proliferation. Heterogeneous levels of cellular stress and noisy signaling processes can lead to phenotypic variability but little is known about their role in underlying proliferation heterogeneity. We study two previously published single cell datasets and find that cells encode heterogeneous levels of endogenous and exogenous DNA damage to shape proliferation heterogeneity at the population level. Using a comprehensive time series analysis of short- and long-term signaling dynamics of p53 and p21, we show that DNA damage levels are quantitatively translated into p53 and p21 signal parameters in a gradual manner. Analyzing instantaneous proliferation and signaling differences among equally-radiated cells, we identify time-localized changes in the period of p53 pulses that drive cells out of a low proliferative state. Our findings suggest a novel role of the p53-p21 network in quantitatively encoding DNA damage strength and fine-tuning proliferation trajectories.</p>
            <img src="conference.png" alt="conference" class="medium-img">
        </div>
        <div class="project-item">
            <h3>3rd German Cancer Research Congress</h3>
            <p><em>10.2023. Heidelberg, Germany</em></p>
            <p><strong>Scientific talk on DNA damage response shaping proliferation</strong></p>
            <p>Analysis of the signaling dynamics of the tumor suppressor p53 and its downstream target, p21, using live cell imaging, time series analysis, and machine learning algorithms to quantify DNA damage responses at the single-cell level. These findings suggest a novel role of the p53-p21 network in quantitatively encoding DNA damage strength and fine-tuning proliferation behavior.</p>
        </div>
        <div class="project-item">
            <h3>e:Med Meeting on Systems Medicine</h3>
            <p><em>11.2022. Heidelberg, Germany</em></p>
            <p><strong>Poster on DNA damage response in single cells</strong></p>
        </div>
        <div class="project-item">
            <h3>EMBO/EMBL Symposium: Biological Oscillators</h3>
            <p><em>03.2022. Heidelberg, Germany</em></p>
            <p><strong>Poster on single-cell p53 and p21 signaling dynamics and DNA damage</strong></p> 
        </div>
    </section>

    <section id="skills">
        <h2>SKILLS</h2>
        <div class="project-item">
            <h3>Programming Languages</h3>
            <ul>
                <li>Python</li>
                <li>MATLAB</li>
                <li>R</li>
                <li>Bash</li>
                <li>FORTRAN</li>
                <li>JAVA</li>
                <li>HTML</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>Data Analysis & Bioinformatics Tools</h3>
            <ul>
                <li>ilastik</li>
                <li>ImageJ/Fiji</li>
                <li>CellProfiler</li>
                <li>MetaXpress</li>
                <li>IncuCyte</li>
                <li>Elasticsearch</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>Operating Systems</h3>
            <ul>
                <li>OS-X</li>
                <li>Windows</li>
                <li>Linux</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>High-Performance Computing (HPC)</h3>
            <ul>
                <li>Proficient with HPC resources, parallel computing, and job scheduling.</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>Technologies</h3>
            <ul>
                <li>Github</li>
                <li>Docker</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>Python Packages</h3>
            <ul>
                <li>Data Science: NumPy, pandas, SciPy, Matplotlib, seaborn</li>
                <li>Bioinformatics: Biopython, scikit-bio</li>
                <li>Machine Learning: scikit-learn, TensorFlow, Keras, PyTorch, XGBoost</li>
                <li>Biostatistics: statsmodels, lifelines</li>
                <li>Visualization: Plotly, Bokeh</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>MATLAB Toolboxes</h3>
            <ul>
                <li>Data Science: Statistics and Machine Learning Toolbox, Deep Learning Toolbox</li>
                <li>Bioinformatics: Bioinformatics Toolbox</li>
                <li>Computational Biology: Systems Biology Toolbox, SimBiology</li>
                <li>Visualization: MATLAB Graphics, MATLAB Plot Gallery</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>R Packages</h3>
            <ul>
                <li>Data Science: dplyr, tidyr, readr, data.table</li>
                <li>Bioinformatics: Bioconductor, GenomicRanges</li>
                <li>Machine Learning: caret, randomForest, xgboost</li>
                <li>Biostatistics: survival, survminer, rms</li>
                <li>Visualization: ggplot2, lattice, plotly</li>
            </ul>
        </div>
        <div class="project-item">
            <h3>Others</h3>
            <ul>
                <li>Scientific writing</li>
                <li>Public speaking</li>
                <li>Presentations</li>
                <li>Mentoring</li>
                <li>Project management</li>
            </ul>
        </div>

    </section>

    <section id="languages">
        <h2>LANGUAGES</h2>
        <ul>
            <li>Spanish: Native</li>
            <li>Catalan: Native</li>
            <li>Moldavian: Native</li>
            <li>Romanian: Native</li>
            <li>English: Fluent</li>
            <li>Russian: Fluent</li>
        </ul>
    </section>

    <section id="contact">
        <h2>CONTACT & MORE INFO</h2>
        <p><a href="mailto:nica.gutu12@gmail.com"><i class="fas fa-envelope"></i>  Email</a></p>
        <p><a href="https://www.linkedin.com/in/nica-gutu" target="_blank"><i class="fab fa-linkedin"></i>  LinkedIn</a></p>
        <p><a href="https://github.com/nica-gutu" target="_blank"><i class="fab fa-github"></i>  GitHub</a></p>
    </section>

    <footer>
        <p>&copy; 2024 Nica Gutu. All Rights Reserved.</p>
    </footer>

    <script>
        function showSection(sectionId) {
            document.querySelectorAll('section').forEach(section => {
                section.classList.remove('active');
            });

            document.querySelectorAll('nav a').forEach(link => {
                link.classList.remove('active');
            });

            document.getElementById(sectionId).classList.add('active');

            document.getElementById('nav-' + sectionId).classList.add('active');
        }

        showSection('about');

    </script>
</body>
</html>
